apps/cowswap-frontend/src/modules/permit/hooks/usePreGeneratedPermitInfo.ts

```diff
@@ -12,7 +12,7 @@ import { PRE_GENERATED_PERMIT_URL } from '../const'
  * Caches result with SWR until a page refresh
  */
 export function usePreGeneratedPermitInfo(): {
-  allPermitInfo: Record<string, PermitInfo>
+  allPermitInfo: Record<string, PermitInfo | undefined>
```

Comment:
Why undefined?
This comes from the github token-lists repo.
It either has the permit info for a token or not at all. `undefined` is not stored.